src/pages/resultsView/ResultsViewPageStore.ts

```diff
@@ -2172,6 +2183,28 @@ export class ResultsViewPageStore {
         }
     });
 
+    readonly referenceGenes = remoteData<ReferenceGenomeGene[]>({
+        await: ()=>[
+            this.studies,
+            this.mutationEnrichmentData,
+            this.copyNumberAmpEnrichmentData,
+            this.genes
+        ],
+        invoke: async () => {
+            const queryGenes = this.genes.result!.map((g:Gene)=>g.hugoGeneSymbol.toUpperCase());
```

Comment:
is there a reason why we have to uppercase them?  seems like not a good practice.

Reply:
we do this everywhere. it is unpleasant but... it is